src/core/components/icon/iconList.ts

```diff
@@ -10,6 +10,7 @@ import BlockchainBlockchain from '../../assets/icons/blockchain-blockchain.svg';
 import BlockchainGasfee from '../../assets/icons/blockchain-gasfee.svg';
 import BlockchainSmartContract from '../../assets/icons/blockchain-smartcontract.svg';
 import BlockchainWallet from '../../assets/icons/blockchain-wallet.svg';
+import BlockChainWalletConnect from '../../assets/icons/blockchain-walletconnect.svg';
```

Comment:
I just saw the icon-name and enum name are not aligned, could you match them? If the asset name is blockchain-walletconnect, the enum name should match it as we're doing for all other icons